SPECS/kube-vip-cloud-provider/kube-vip-cloud-provider.signatures.json

```diff
@@ -0,0 +1,6 @@
+{
+    "Signatures": {
+     "kube-vip-cloud-provider-0.0.2-vendor.tar.gz": "6dda8a2eb13a1c29f5f5c70deaa7f399e6961c221d9c8ee3b0323c9d99ee5a12",
+     "kube-vip-cloud-provider-0.0.2.tar.gz": "f8cfdff062f55c5e167503e1fc1ca0e99f9d53858660a3be07858a3db680a7e5"
```

Comment:
These files do not seem to present in the blob store :)

Reply:
Yes, I will add them in the blob store, by today.

Reply:
Can you try now? :-)

Reply:
We recently did a change that allows anyone to run a `BuddyBuild`.
Can you please include the `BuddyBuild` Pipeline id for both AMD64 & ARM64 for your changes.
Thanks.

> Can you try now? :-)

Sure, this is a representative example for AMD64 BuddyBuild of this PR: [PR-3728](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=236835&view=logs&j=75cba72d-09cb-503a-3927-4ab61ac4a179&t=246631ef-4e63-576d-3b21-bc52cd4b0a93&l=2133)

Please run it similarly for ARM64 [for your other change as well :)]
Also, I added this specific detail in the description of the PR under the testing methodology.

Reply:
Added this. Thanks for the information.